crates/nu-command/src/system/run_external.rs

```diff
@@ -123,15 +152,20 @@ impl Command for External {
         let args = eval_arguments_from_call(engine_state, stack, call)?;
         #[cfg(windows)]
         if is_cmd_internal_command(&name_str) || potential_nuscript_in_windows {
-            use std::os::windows::process::CommandExt;
```

Comment:
I don't think it's necessary to move the line to the top of file

Reply:
If I leave it here, then I also have to put it on line 164. I'd rather just have it in one place in the file.